src/common/sonarqube-v4/helpers/temp-find-method.ts

```diff
@@ -1,4 +1,9 @@
-/* istanbul ignore file */
+/* eslint-disable max-depth */
+/* eslint-disable no-var */
+/* eslint-disable eqeqeq */
+/* eslint-disable prefer-const */
+/* eslint-disable complexity */
+/* eslint-disable no-underscore-dangle */
 /*Can be deleted once https://www.npmjs.com/package/azure-pipelines-task-lib v3.1.0 will be release */
 import * as tl from "azure-pipelines-task-lib/task";
 import path = require("path");
```

Comment:
No idea, [it's been working like this in the past](https://github.com/SonarSource/sonar-scanner-vsts/blob/master/common/ts/helpers/temp-find-method.ts#L4) since this wasn't changed by my PR 

I've changed it in my PR, because you're right for sure this syntax is weird

<details>

Here's a `.yml` I use to test `V4`:

```yml
steps:
- task: SonarQubePrepare@4
  inputs:
    SonarQube: 'SQ (7ph)'
    scannerMode: 'CLI'
    configMode: 'manual'
    cliProjectKey: 'powerglitch'
    cliProjectName: 'powerglitch'
    cliSources: '.'
- task: SonarQubeAnalyze@4
- task: SonarQubePublish@4
  inputs:
    pollingTimeoutSec: '300'
```

If you're using SQ >= 10.4 (requires Java 17) and Microsoft hosted agents:
```yml
variables:
  JAVA_HOME_11_X64: /usr/lib/jvm/temurin-17-jdk-amd64
```
</details>

Reply:
thanks, after setting the variable for java to 17 it worked :D 

easier than getting SQ 10.3 running locally... but it proves the v4 tasks are still functional 👍🏻